eureka2-write-server/src/main/java/com/netflix/eureka2/server/EurekaWriteServerModule.java

```diff
@@ -104,5 +102,7 @@ public void configureEureka() {
         bind(SelfRegistrationService.class).to(EurekaWriteServerSelfRegistrationService.class).asEagerSingleton();
 
         bind(AbstractEurekaServer.class).to(EurekaWriteServer.class);
+
+        install(new OverridesModule());
```

Comment:
We had specific bindings for override stuff that breaks the write server when removed. What we need is some no-op bindings.